modes/gnus/evil-collection-gnus.el

```diff
@@ -192,7 +185,75 @@
 
   (evil-set-initial-state 'gnus-article-mode 'normal)
   (evil-define-key 'normal gnus-article-mode-map
-    "q" 'evil-window-delete))
+    "r" 'gnus-summary-reply
+    "R" 'gnus-summary-reply-with-original
+    "q" 'evil-window-delete)
+
+  (evil-set-initial-state 'gnus-group-mode 'normal)
+  (evil-define-key 'normal gnus-group-mode-map
+
+    ;; quit
+    "q"  'gnus-group-exit
+    "ZZ" 'gnus-group-exit
+    "ZQ" 'gnus-group-quit
+
+    "x" 'gnus-group-kill-group
+    "p" 'gnus-group-yank-group
+
+    ; Marking
+    "m" 'gnus-group-mark-group
+    "u" 'gnus-group-unmark-group
+    "U" 'gnus-group-unmark-all-groups
+    "M" 'gnus-group-mark-buffer
+    "*" 'gnus-group-mark-buffer
+    "%" 'gnus-group-mark-regexp
+
+    ; Searching
+    "s" 'gnus-group-apropos
+    "S" 'gnus-group-description-apropos
+
+    "^" 'gnus-group-enter-server-mode
+
+    (kbd "RET") 'gnus-group-select-group
+    "g?" 'gnus-group-help-map
+    "a" 'gnus-group-mail)
+
+  (evil-set-initial-state 'gnus-server-mode 'normal)
+  (evil-define-key 'normal gnus-server-mode-map
+    (kbd "RET") 'gnus-server-read-server
+    (kbd "SPC") 'gnus-server-read-server-in-server-buffer
+    "C"         'gnus-server-close-server
+    "D"         'gnus-server-deny-server
+    "G"         'gnus-group-make-nnir-group
+    "I"         'gnus-server-set-cloud-method-server
+    "L"         'gnus-server-offline-server
+    "O"         'gnus-server-open-server
+    "R"         'gnus-server-remove-denials
+    "S"         'gnus-server-show-server
+    "a"         'gnus-server-add-server
+    "y"         'gnus-server-copy-server
+    "e"         'gnus-server-edit-server
+    "g"         'gnus-server-regenerate-server
+    "h"         'describe-mode
```

Comment:
Avoid binding to `hjkl` it can be frustrating.  Prefer `g?` for this one.

Reply:
My bad! That one slipped through copy pasted from the original bindings. It's one of the lesser used modes so I wasn't bugged by it yet. Thanks for catching that!